pkg/controllers/cassandra/cluster_control.go

```diff
@@ -25,22 +26,35 @@ type ControlInterface interface {
 var _ ControlInterface = &defaultCassandraClusterControl{}
 
 type defaultCassandraClusterControl struct {
-	recorder record.EventRecorder
+	serviceControl service.Interface
+	recorder       record.EventRecorder
 }
 
-func NewControl(recorder record.EventRecorder) ControlInterface {
+func NewControl(serviceControl service.Interface, recorder record.EventRecorder) ControlInterface {
 	return &defaultCassandraClusterControl{
-		recorder: recorder,
+		serviceControl: serviceControl,
+		recorder:       recorder,
 	}
 }
 
 func (e *defaultCassandraClusterControl) Sync(c *v1alpha1.CassandraCluster) error {
 	glog.V(4).Infof("defaultCassandraClusterControl.Sync")
+	err := e.serviceControl.Sync(c)
+	if err != nil {
+		e.recorder.Eventf(
+			c,
+			"cassandra.defaultCassandraClusterControl",
+			ErrorSync,
+			MessageErrorSyncService,
+			c,
+		)
+		return err
+	}
 	e.recorder.Event(
 		c,
 		"cassandra.defaultCassandraClusterControl",
```

Comment:
Again, this should be `v1.EventTypeNormal`. We shouldn't be just making these up 😄

Reply:
Done.